static/js/webrtc.js

```diff
@@ -182,8 +184,24 @@ var rtc = (function() {
     toggleVideo: function() {
       var videoTrack = localStream.getVideoTracks()[0];
       if (videoTrack) {
-        videoTrack.enabled = !videoTrack.enabled;
-        return !videoTrack.enabled;
+        if (videoTrack.enabled) {
+          videoTrack.enabled = false;
+          videoTrack.stop()
+          return false
+        } else {
+          var audioTrack = localStream.getAudioTracks()[0];
+          console.log(1, audioTrack)
+          var audioTrackEnabled = audioTrack.enabled;
+          self.deactivate()
+          self.activate()
+          .then(() => {
+            // new one, now that we've reactivated
+            audioTrack = localStream.getAudioTracks()[0];
```

Comment:
Assuming I do re-activate, I have to return audio to the same state it was at before. Though, I don't have a lot of confidence that this works.